OrbitGl/App.h

```diff
@@ -233,6 +233,14 @@ class OrbitApp final : public DataViewFactory, public CaptureListener {
 
   DataView* GetOrCreateDataView(DataViewType type) override;
 
+  const std::unique_ptr<ProcessManager>& GetProcessManager() {
+    return process_manager_;
+  }
+  const std::unique_ptr<ThreadPool>& GetThreadPool() { return thread_pool_; }
+  const std::unique_ptr<MainThreadExecutor>& GetMainThreadExecutor() {
```

Comment:
return a pointer - we do not transfer ownership here, also [[nodiscard]]

Reply:
Done for the pointer part. Let's discuss the use of [[nodiscard]] with the team.

Reply:
Done (nodiscard).